core/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/CollectorService.scala

```diff
@@ -243,7 +243,7 @@ class CollectorService(
     userAgent.foreach(e.userAgent   = _)
     refererUri.foreach(e.refererUri = _)
     e.hostname      = hostname
-    e.networkUserId = networkUserId
+    e.networkUserId = anonymize(networkUserId, spAnonymous).orEmpty
```

Comment:
Given that it's the only place where the function is used - I think we can just return empty string?

Or even abandon the function altogether:

```suggestion
    e.networkUserId = spAnonymous.fold(networkUserId)(_ => "")
```

Reply:
Nice. I started with the function but since it's the only usage this makes sense to tidy up.